tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -1615,6 +1615,29 @@ describe("Spreadsheet Pivot", () => {
     );
   });
 
+  test("Pivot with date and datetime measures", () => {
+    const model = createModelFromGrid({
+      A1: "Date",
+      B1: "Datetime",
+      A2: "2024/02/03",
+      B2: "2024/02/03 12:34:56",
+      A3: "2022/04/14",
+      B3: "2022/04/14 01:02:03",
+    });
+    addPivot(model, "A1:B3", {
+      columns: [],
+      rows: [],
+      measures: [
+        { id: "Date:max", fieldName: "Date", aggregator: "max" },
+        { id: "Datetime:max", fieldName: "Datetime", aggregator: "max" },
```

Comment:
nitpick: could test differents aggregator rather than 2 max to be a bit more complete